### PR TITLE
CMakeLists.txt update to remove unnecessary variable

### PR DESCRIPTION
**Pull request by hansely**
Remove unnecessary variable : MIVisionX_LIBRARIES_DIRS
Update rocAL CMake to exclude rocAL_pybind subdirectory if manually turned off

### CODE REVIEW

**Review by rrawther — APPROVED** [review]

File: CMakeLists.txt [hunk 1]
```diff
@@ -49,11 +50,8 @@ endif(CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
 # rocAL Default Options
 option(ENHANCED_MESSAGE "rocAL Enhanced Message Option"        ON)
 option(GPU_SUPPORT      "Build rocAL with GPU Support"         ON)
-option(BUILD_DEV        "Build rocAL Development Package"      ON)
-option(AMD_FP16_SUPPORT "Build rocAL with float16 Support"    OFF)
 option(BUILD_PYPACKAGE  "Build rocAL Python Package"           ON)
-option(BUILD_WITH_AMD_ADVANCE "Build rocAL for advanced AMD GPU Architecture"    OFF)
-option(PYTHONVERSION    "Python version to build rocal" "")
+option(BUILD_WITH_AMD_ADVANCE  "Build rocAL with Advanced GPU support" OFF)
```
Comment (LakshmiKumar23) [inline CMakeLists.txt:54]: @hansely why was pythonversion removed? Seems like you have not synced with ToT.

File: rocAL_pybind/CMakeLists.txt [hunk 2]
```diff
@@ -32,22 +34,16 @@ set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/${CMAKE_INSTALL_LIBDIR})
...
 find_package(MIVisionX QUIET)
 find_package(TurboJpeg QUIET)
-if (PYTHONVERSION)
-    find_package(Python3 "${PYTHONVERSION}" EXACT QUIET COMPONENTS Interpreter Development)
-else()
-    set(Python3_FIND_VIRTUALENV FIRST)
-    find_package(Python3 QUIET COMPONENTS Interpreter Development)
-endif()
-find_package(pybind11 QUIET CONFIG)
+set(Python3_FIND_VIRTUALENV FIRST)
```
Comment (LakshmiKumar23) [inline rocAL_pybind/CMakeLists.txt:44]: @hansely Please undo these changes and sync to ToT

**Comment by hansely** [comment]
Closing this PR and reopening with develop branch synced
